src/views/Home.jsx

```diff
@@ -1,8 +1,38 @@
 import './Home.css';
 import { SingleList } from '../components';
+import { useState } from 'react';
+import { createList, useAuth } from '../api';
+import { useNavigate } from 'react-router-dom';
 
 export function Home({ data, setListPath }) {
 	const hasList = data.length !== 0;
+
+	const { user } = useAuth();
+	const userId = user?.uid;
+	const userEmail = user?.email;
+
+	const [inputValue, setInputValue] = useState('');
+	const navigate = useNavigate();
+
+	const handleSubmit = async (e) => {
+		e.preventDefault();
+
```

Comment:
We might want to `trim` the `inputValue` and ensure it still has a length > 0 before creating the list. We currently allow for a single space character to be used as the list name

![image](https://github.com/user-attachments/assets/3c7e1740-9411-42e0-a790-ccb80d2e5458)

Reply:
That is a great callout for form validations! 😅 Definitely don't wan't garbage data going into the database.